### PR TITLE
Customized the Open MatyerialX Document dialog: *.mtlx file filter, s…

### PR DESCRIPTION
**Pull request by ppenenko**
…tarting in the directory of the previous path if possible.

### CODE REVIEW

File: source/MaterialXContrib/MaterialXMaya/AEtemplate/AEMaterialXSurfaceTemplate.mel [hunk 1]
```diff
@@ -1,27 +1,65 @@
...
+global proc MaterialXSurface_FilePathReplace(string $attrName)
+{
+    evalDeferred ("connectControl documentFilePathField " + $attrName);
+    button -edit -command ("MaterialXSurface_FileBrowser " + $attrName) documentFileBrowserButton;
+}
+
+global proc MaterialXSurface_FileBrowser(string $attrName) 
+{
+    // Start in current file's directory if possible. 
+	string $sStartingdir = `textField -q -fileName documentFilePathField`;
```
Comment (feldstj) [inline source/MaterialXContrib/MaterialXMaya/AEtemplate/AEMaterialXSurfaceTemplate.mel]: looks like you have a whitespace issue here.

Comment (ppenenko) [reply source/MaterialXContrib/MaterialXMaya/AEtemplate/AEMaterialXSurfaceTemplate.mel]: That's right, copied-and-pasted from another plugin. Fixed now.

File: source/MaterialXContrib/MaterialXMaya/AEtemplate/AEMaterialXSurfaceTemplate.mel [hunk 2]
```diff
@@ -1,27 +1,65 @@
-global proc MaterialXSurface_extraCtrlsBuild(string $nodeName)
```
Comment (ppenenko) [inline source/MaterialXContrib/MaterialXMaya/AEtemplate/AEMaterialXSurfaceTemplate.mel]: "Extra controls" has a special meaning in AE templates - as in "editorTemplate -addExtraControls".

**Comment by ppenenko** [comment]
AppVeyor failure looks random and unrelated.

**Pull request merged**